botwo/botwo.py

```diff
@@ -1,131 +1,144 @@
 import fnmatch
+import botocore
 
-SPECIAL_METHODS = {"get_paginator", "delete_objects"}
 COPY_METHODS = {"copy", "copy_object", "copy_upload_part"}
+LIST_METHODS = {"list_objects", "list_objects_v2", "list_object_version"}
 
 
-def client(client_mapping, config):
-    botwo = Botwo()
-    return botwo.create_client_class(client_mapping, config)
+class PaginatorWrapper(object):
+    """ This class wraps boto paginator """
 
-
-class Paginatwo(object):
-
-    def __init__(self, mapping, kwargs, client1, client2, client1_name, client2_name):
+    def __init__(self, mapping, config, kwargs):
```

Comment:
I don't understand this usage of "`kwargs`".  I don't think I have ever seen it in an arg list as anything except "`**kwargs` -- when its usage becomes apparent.

Reply:
This is still relevant.  "`kwargs`" used like this is confusing.

Reply:
Done